pkg/build/builder/daemonless.go

```diff
@@ -259,6 +259,11 @@ func buildDaemonlessImage(sc types.SystemContext, store storage.Store, isolation
 		transientMounts = append(transientMounts, "/run/secrets:/run/secrets:ro,nodev,noexec,nosuid")
 	}
 
+	seccompProfilePath := "/usr/share/containers/seccomp.json"
+	if profileEnv, ok := os.LookupEnv("BUILD_SECCOMP_PROFILE"); ok {
+		seccompProfilePath = profileEnv
+	}
+
```

Comment:
@nalind per my comment on how to default, would this change to add teh seccomp only be done if we were OCI isolation ?

Reply:
The seccomp filter is also applied when using chroot isolation.  The magic value "unconfined" is recognized as instructing us to not add a filter.

Reply:
turns out `BUILD_SECCOMP_PROFILE` is not an env var in the approved list at https://github.com/openshift/api/blob/master/build/v1/consts.go#L169

similar to the removing the case statement on the isolation, let remove the if check and just set seccompProfilePath to the default location.

Reply:
Changed it.